py3status/modules/mpd_status.py

```diff
@@ -220,7 +220,7 @@ def attr_getter(attr):
             c.disconnect()
 
         if len(text) > self.max_width:
-            text = text[:-self.max_width - 3] + '...'
+            text = text[:self.max_width - 3] + '...'
```

Comment:
I'd prefer `text = u'{}...'.format(text[:self.max_width - 3])` as I'm worried the original may not be unicode safe in python2